src/pages/profile.tsx

```diff
@@ -454,14 +456,13 @@ const Profile: NextPage = () => {
                       />
                     </div>
                   </div>
+                  <Note className="md:w-96">
+                    If you don&apos;t have set times, communicate that on your
+                    own with potential riders/drivers. For start/end time, enter
+                    whatever best matches your work schedule.
+                  </Note>
                   <div className="flex flex-col space-y-2">
-                    <div className="flex flex-row items-center">
-                      <Note>
-                        If you don&apos;t have set times, communicate that on
-                        your own with potential riders/drivers. For start/end
-                        time, enter whatever best matches your work schedule.
-                      </Note>
-                    </div>
+                    <div className="flex flex-row items-center"></div>
```

Comment:
Is there a reason why this div is here if it has nothing in it?

Reply:
Yeah just missed this while moving things around, I'll remove this